library/src/main/java/com/getstream/sdk/chat/view/ChannelHeaderViewStyle.java

```diff
@@ -50,6 +51,7 @@ public ChannelHeaderViewStyle(Context c, AttributeSet attrs) {
         lastActiveTextSize = (int) a.getDimension(R.styleable.ChannelHeaderView_streamChannelHeaderLastActiveTextSize, getDimension(R.dimen.stream_channel_preview_date));
         lastActiveTextColor = a.getColor(R.styleable.ChannelHeaderView_streamChannelHeaderLastActiveTextColor, getColor(R.color.stream_gray_dark));
         lastActiveTextStyle = a.getInt(R.styleable.ChannelHeaderView_streamChannelHeaderLastActiveTextStyle, Typeface.NORMAL);
+        offlineText = a.getString(R.styleable.ChannelHeaderView_streamChannelHeaderOfflineTextStyle);
```

Comment:
`ChannelHeaderView_streamChannelHeaderOfflineTextStyle` looks weird name.
Why doesn't it have `ChannelHeaderView_streamChannelHeaderOfflineText`?